### ISSUE TITLE
Fake Tensor Enablement Follow - Ups

### ISSUE THREAD

**Issue by eellison** [pytorch/pytorch#93669]
- [ ] fix batchnorm 2d decomp cpu error 

model failures: 
- [x] "hf_Bert", 
- [x] "drq" (fixed with https://github.com/pytorch/pytorch/pull/85065)
- [x] "soft_actor_critic" (fixed with https://github.com/pytorch/pytorch/pull/85065)
- [x] "dpn107" (Fixed with https://github.com/pytorch/pytorch/pull/85065)

cc @ezyang @soumith @msaroufim @wconstab @ngimel @bdhirsh

**Comment by ezyang** [comment]
guessing this done
